doc/source/configuration/wazuh.rst

```diff
@@ -0,0 +1,435 @@
+
+
+Edit the playbook and variables to your needs:
+
+Wazuh manager configuration
+---------------------------
+
+Wazuh manager playbook is located in ``etc/kayobe/ansible/wazuh-manager.yml``.
+Running this playbook will:
+
+* generate certificates for wazuh-manager
+* setup and deploy filebeat on wazuh-manager vm
+* setup and deploy wazuh-indexer on wazuh-manager vm
+* setup and deploy wazuh-manager on wazuh-manager vm
+* setup and deploy wazuh-dashboard on wazuh-manager vm
+* copy certificates over to wazuh-manager vm
+
+Wazuh manager variables file is located in ``etc/kayobe/inventory/group_vars/wazuh-manager/wazuh-manager``.
+
+You may need to modify some of the variables, including:
+
+* domain_name
+* wazuh_manager_ip
+* private_ip
+
+.. note::
+
+    NOTE:
+    If you are using multiple environments you will need to move variable files to pertinent directory,
+    for example `~/deployment/src/kayobe-config/etc/kayobe/environments/production/inventory/group_vars/`
+    Files to be moved:
+    - etc/kayobe/inventory/group_vars/wazuh/wazuh-manager/wazuh-manager
+    - inventory/group_vars/wazuh/wazuh-manager/wazuh-secrets
+    - etc/kayobe/inventory/group_vars/wazuh/wazuh-agent/wazuh-agent
+
+Secrets
+=======
+
+Wazuh secrets playbook is located in ``etc/kayobe/ansible/wazuh-secrets.yml``.
+Running this playbook will generate and put pertinent security items into secrets
+vault file which will be placed in ``$KAYOBE_CONFIG_PATH/wazuh-secrets.yml``.
+Remember to encrypt!
+
+Wazuh secrets template is located in ``etc/kayobe/ansible/templates/wazuh-secrets.yml.j2``.
+It will be used by wazuh secrets playbook to generate wazuh secrets vault file.
+
+
+.. code-block:: console
+
+  kayobe playbook run $KAYOBE_CONFIG_PATH/ansible/wazuh-secrets.yml -e wazuh_user_pass=$(uuidgen) -e wazuh_admin_pass=$(uuidgen)
+  ansible-vault encrypt --vault-password-file ~/vault.pass $KAYOBE_CONFIG_PATH/inventory/group_vars/wazuh/wazuh-manager/wazuh-secrets
+
+
+==============
+TLS (optional)
+==============
+
+You can generate your own TLS certificates, otherwise skip this section.
+By default, Wazuh Ansible uses `wazuh-cert-tool.sh <https://documentation.wazuh.com/current/user-manual/certificates.html>`__
+to automatically
+generate certificates for wazuh-indexer (previously Elasticsearch and opendistro)
+and wazuh-dashbooard (previously Kibana) using a local CA.
+If the certificates directory ``etc/kayobe/ansible/wazuh/certificates``
+does not exist, it will generate the following certificates in ``etc/kayobe/ansible/wazuh/certificates/certs/``
+(here os-wazuh is set as ``elasticsearch_node_name`` and ``kibana_node_name``:
+
+
+* Admin certificate for opendistro security
+   * admin.key,  admin.pem
+* Node certificate
+   * os-wazuh.key,  os-wazuh.pem
+* HTTP certificate for Kibana (port 5601) & Elasticsearch (port 9200)
+   * os-wazuh_http.key, os-wazuh_http.pem
+* Root CA certificate
+   * root-ca.key  root-ca.pem
```

Comment:
It appears the [indexer role](https://github.com/stackhpc/wazuh-ansible/blob/970263ae2e078c63d71b9e89643fbb231c754d5a/roles/wazuh/wazuh-indexer/tasks/security_actions.yml#L29-L34) has renamed some of these keys for example `admin.key` is `admin-key.pem`

```suggestion
* Admin certificate for opendistro security
   * admin-key.pem,  admin.pem
* Node certificate
   * os-wazuh-key.pem,  os-wazuh.pem
* HTTP certificate for Kibana (port 5601) & Elasticsearch (port 9200)
   * os-wazuh_http.key, os-wazuh_http.pem
* Root CA certificate
   * root-ca.key  root-ca.pem
```

Reply:
Fixed. Thanks for commit suggestion!